MinecraftClient/Commands/Inventory.cs

```diff
@@ -9,7 +9,7 @@ namespace MinecraftClient.Commands
     class Inventory : Command
     {
         public override string CMDName { get { return "inventory"; } }
-        public override string CMDDesc { get { return "inventory <<id>|player|container> <list|close|click <slot>>: Interact with inventories"; } }
+        public override string CMDDesc { get { return "inventory <<id>|player|container> <list|close|click <slot> <L|R|M>>: Interact with inventories"; } }
```

Comment:
Although it requires more characters to type,  `<L|R|M>` may be unclear especially in scripts, and given that this command is already pretty complex. I think it should be documented as `<left|right|middle>`. You can optionally add support for `l` `r` `m` by only checking the first character, e.g. document `<left|right|middle>` but check if `b.ToLower()[0] == "r"` instead of `b.ToLower() == "right"`.